### PR TITLE
Add Monster Hunter Generations

### PR DESCRIPTION
**Pull request by Filxter**
Adds Monster Hunter Generations.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by robzombie91** [comment]
Images are being detected as .jpg instead of .png. Try reuploading them by opening them in paint or any image manipulation program and save it as .png

**Comment by Filxter** [comment]
Where should i go to reupload the images?

**Comment by Filxter** [comment]
Boxart reuploaded and everything seems to be in perfect order.

File: games/monster-hunter-generations/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Monster Hunter Generations"
+description = "Monster Hunter Generations is an action role-playing video game developed and published by Capcom for the Nintendo 3DS."
```
Comment (Leo626) [inline games/monster-hunter-generations/game.dat]: There should be a mention that the game is called Monster Hunter X in Japan and that there is a Japan only re-release. I added this to the description.

> Monster Hunter Generations, known as Monster Hunter X in Japan, is an action role-playing video game developed and published by Capcom for the Nintendo 3DS. An expanded version of the game called Monster Hunter XX: Double Cross was released only in Japan.

**Comment by Leo626** [comment]
Add the japan re-release.

`0004000000197100` - `2017-03-18`

**Comment by Filxter** [comment]
Description updated and japan re-release added.

**Pull request merged**